### ISSUE TITLE
Many TS source files link to non-existing documentation

### ISSUE THREAD

**Issue by mikkorantalainen** [hugerte/hugerte#29]
The rebranding rename (https://github.com/hugerte/hugerte/commit/c0d15c424d64224a7f5ad28bef581300c7ec0891) accidentally broke links pointing to `tiny.cloud` web site. Some of these URLs are visible to the end user.

Here's a patch that reverts the change for those URLs:
https://github.com/mikkorantalainen/hugerte/commit/0cddc170b4268a22bbb9feeff735b13010d9a6d7

I created the patch with
```sh
find -type f -not -path '*/.git/*' -exec perl -i -pe 's@tiny[.]cloud/docs/hugerte/@tiny.cloud/docs/tinymce/@g' {} +
```
and manually reviewed the results. I don't know if rebranding also broke some other URLs but these are the only ones I'm aware of.

**Comment by mikkorantalainen** [comment]
The commit message should be titled as "Partially revert the new branding which accidentally broke some URLs" instead of having word "rename" in there.

**Comment by carlosmintfan** [comment]
Thank you very much! I'll fix this stuff in about a week. I'm thinking about going the opposite direction: finally making the links go to hugerte.org directly and linking to the TinyMCE docs there, or putting the docs on the HugeRTE website – I'll just have to setup Antora and find a good theme for it.

**Comment by carlosmintfan** [comment]
See https://github.com/hugerte/hugerte-website/commit/0b13fd6b0a4cf90bce9a85c6ed7498f3dc26dc5b, I've decided against forking the old docs, though you may tell me if you think that's a bad idea.

**Comment by carlosmintfan** [comment]
Sorry for the delay. I now did this (modified your command):
```bash
find -type f -not -path '*/.git/*' -exec perl -i -pe 's@tiny[.]cloud/docs/hugerte/@hugerte.org/docs/hugerte/@g' {} +
```
Then replaced `hugerte/6` by `hugerte/1` using VSCode search and replace.
I'll publish a new release today.

**Comment by carlosmintfan** [comment]
Fixed by 1725d8fe1f.
